html/rendering/non-replaced-elements/phrasing-content-0/WEB_FEATURES.yml

```diff
@@ -2,3 +2,6 @@ features:
 - name: b
   files:
   - b-strong-styles.html
+- name: strong
```

Comment:
Both b and strong are mapped to the same test. I think this may be unavoidable in the current state though?